src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+$body-font: "Roboto", sans-serif;
+$font-size: 10px;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-blue-accent: #00acdc;
+
 body {
   margin: 0;
+
+  font-family: $body-font;
+  font-size: $font-size;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
It would be better to set max-width: 200px and width: 100%; so the card can shrink if needed.